bigtop-packages/src/common/apex/do-component-build

```diff
@@ -28,6 +28,9 @@ command -v mvn > /dev/null 2>&1 || {
 # Build apex
 mvn clean package -DskipTests "$@"
 
+#Find hadoop.version from pom.xml
+APEX_HADOOP_VERSION=$(grep -oPm1 "(?<=<hadoop.version>)[^<]+" pom.xml)
```

Comment:
> I would also like to point out that the Apex package can be assembled without building Apex from source, by optionally using the official release
We actually _always_ build all the components from scratch. That's the policy aimed to guarantee the compatibility through the stack.